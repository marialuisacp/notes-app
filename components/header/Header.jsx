import React from 'react';
import StyledHeader from './styles/Header.styles';
import HeaderButton from '../headerButton/HeaderButton';

const Header = () => (
  <StyledHeader>
    <span> Teste </span>
    <HeaderButton icon="application-switcher"></HeaderButton>
  </StyledHeader>
);

export default Header;