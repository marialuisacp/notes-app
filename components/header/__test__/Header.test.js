import React from 'react';
import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import Header from '../Header';

test('should render component Header', () => {
  const { HeaderFragment } = render(<Header></Header>);
  expect(HeaderFragment).toMatchSnapshot();
})
