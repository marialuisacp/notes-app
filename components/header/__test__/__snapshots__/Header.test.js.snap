// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component Header 1`] = `undefined`;
